css/_font.scss

```diff
@@ -25,6 +25,9 @@
     -moz-osx-font-smoothing: grayscale;
 }
 
+.icon-chat1:before {
```

Comment:
Why do we need a new chat icon? The web app uses a chat icon already. Why not use the same?

Reply:
This is for the case when we have unread messages, as we don't use badges on mobile (or at least not yet)

Reply:
But we also change the background color. Isn't that enough?

Reply:
I felt like that's not obvious enough (please see it yourself, I think you'll see) so I added an icon change too. Now it looks really good.

On the other hand, it's just an initial version, we will surely improve it later

Reply:
Let's discuss it together with styling.